### PR TITLE
Refactored the naming convention to BEM and Fixed the footer height.

### PR DESCRIPTION
**Pull request by eaprestige**

### CODE REVIEW

File: src/components/Footer/index.jsx [hunk 1]
```diff
@@ -12,14 +12,16 @@ import logo from '../../assets/images/africai.png';
...
+    <div className="footer-container">
+      <div className="footer-subcontainer">
+        <Link to="/" className="brand-logo">
+          <img src={logo} alt="Africa I" className="footer-logo" />
+        </Link> 
       </div>
-      <div className="col">
-        <h4 className="sub_heading">Quick links</h4>
-        <ul className="links__list">
+      <div className="footer-subcontainer">
```
Comment (eliemugenzi) [inline src/components/Footer/index.jsx:22]: @eaprestige  is this `BEM` format? I think it should be like `footer__subcontainer`. BEM uses double underscores for element and double dashes for a modifier.

File: src/components/Footer/index.jsx [hunk 2]
```diff
@@ -12,14 +12,16 @@ import logo from '../../assets/images/africai.png';
...
+  <footer className="footer">
+    <div className="footer-container">
+      <div className="footer-subcontainer">
+        <Link to="/" className="brand-logo">
+          <img src={logo} alt="Africa I" className="footer-logo" />
+        </Link> 
       </div>
-      <div className="col">
-        <h4 className="sub_heading">Quick links</h4>
-        <ul className="links__list">
```
Comment (eliemugenzi) [inline src/components/Footer/index.jsx:22]: This was in the right BEM format.

File: src/components/Footer/index.jsx [hunk 3]
```diff
@@ -40,9 +42,9 @@ const Footer = () => (
           </li>
         </ul>
       </div>
-      <div className="col">
-        <h4 className="sub_heading">Follow us</h4>
-        <ul className="inline">
+      <div className="footer-subcontainer">
+        <h4 className="sub-heading">Follow us</h4>
+        <ul className="footer-subcontainer-list inline">
```
Comment (eliemugenzi) [inline src/components/Footer/index.jsx:47]: The same applies here

File: src/components/Footer/index.scss [hunk 4]
```diff
@@ -1,69 +1,73 @@
...
 
-  ul.links__list{
-      list-style: none;
-  }
+  
 
-  a{
-      text-decoration: none;
-  }
+
```
Comment (eliemugenzi) [inline src/components/Footer/index.scss:72]: Add a newline at the end of this file

File: src/components/Footer/index.jsx (hunk 1 above)
Comment (eaprestige) [reply src/components/Footer/index.jsx:22]: Okay i will fix that right away @eliemugenzi .

Comment (eliemugenzi) [reply src/components/Footer/index.jsx:22]: Good, keep it up.

**Comment by eliemugenzi** [comment]
You also have merge conflicts.

**Pull request merged**